letsencrypt-win-simple/Plugin/IISPlugin.cs

```diff
@@ -54,7 +54,7 @@ public override List<Target> GetTargets()
  <configuration>
      <system.webServer>
          <staticContent>
-             <mimeMap fileExtension = "".*"" mimeType=""text/json"" />
+             <mimeMap fileExtension = ""."" mimeType=""text/json"" />
```

Comment:
Better add a 
```
<remove fileExtension="." />
```
before adding the mimeMap so it will also work on systems that already have a mime type for extensionless defined.
Then it should work universally.